### ISSUE TITLE
'./assets/fonts/Antwerpen_Icons.eot' is missing in build

### ISSUE THREAD

**Issue by e2o** [a-ui/acpaas_branding_scss#6]
Trying to include `@a-ui/acpaas/dist/main.css` after installing gives the following error:

```
Can't resolve './assets/fonts/Antwerpen_Icons.eot' in '[...]/node_modules/@a-ui/acpaas/dist'
```

It looks like this `fonts/`-folder is missing in the build.

**Comment by e2o** [comment]
I'm on @a-ui/acpaas@2.0.0 btw.

**Comment by TriangleJuice** [comment]
https://github.com/a-ui/acpaas_branding_scss/pull/7 fixes #6
